lib/constants/theme/themes.dart

```diff
@@ -13,6 +13,7 @@ abstract class Theme {
 
   static ThemeContainer<ThemeData> app = ThemeContainer(
     light: ThemeData(
+      useMaterial3: false,
```

Comment:
Flutter 3.16.0 switched the default value of `useMaterial3` to `true` (see [PR](https://github.com/flutter/flutter/pull/129724)). Eventually, we want to switch to Material3, but it changes the theming and the way text themes are inherited, which causes layout and theme changes in the app (see [goldens diffs from before setting the property back to `false`](https://github.com/Abestanis/sweyer/commit/3119509e9135d72e9b82491bcf241f64a096f5a3)). Because I want to keep the diff small, I decided to leave switching to Material3 to a dedicated pull request.

Reply:
> Because I want to keep the diff small, I decided to leave switching to Material3 to a dedicated pull request.

I am not sure if we even want to do this, the whole app design is currenly more about material 2 (and it seems OK honestly)